.changeset/nasty-islands-sin.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/perseus": minor
+---
+
+Fixup scrollbar overflow for mathjax
```

Comment:
I think our future selves might be helped by adding some detail to this description. Maybe something like:

```suggestion
Ensure that radio widget choices can scroll horizontally, but not vertically.
Previously, unnecessary vertical scrollbars appeared on some choices containing
MathJax elements.
```